public/signup.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>signup page</title>
+    <link rel="stylesheet" href="css/signup.css">
+    <link href='https://s.googleapis.com/css?family=Roboto' rel='stylesheet'>
+
+  </head>
+  <body>
+    <div class="main">
+    <header class="header">
+    </header>
+    <div class="container">
+<img src="./image/logo.png" alt="">
+<h1 class="h1">Creat New Account</h1>
+    <form class="form" action="/signup" method="post" id='registerForm'>
+      <input class="input" type="text" id ='username' placeholder="Username" required>
+      <input class="input" type="email" id ='email' placeholder="Email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" required>
+      <input class="input" type="password" id ='password' placeholder="Password" pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}" required>
+      <div class="checkbox"><input  type="checkbox" id="chkOwner" onclick="ShowHideDiv(this)"> Are you shop owner?</div>
+      <div id="divchkOwner" class="divchkOwner" style="display: none">
+        <input class="input" type="text" id ='shopname' placeholder="Shopname" required>
+        <input class="input" type="text" id ='address' placeholder="Address" required>
+      </div>
+      <button class="button" type="submit" name="button" id="btn">Register</button>
+      <h4 class="h4">Already have an account? <a class="a" href="/">Login</a></h4>
+      <div id="paragraph">
+      </div>
+    </form></div>
+  <footer class="footer">2018 Elegant.com © All Rights Reserved</footer>
+</div>
+  </body>
+  <script type="text/javascript">
+  function ShowHideDiv(chkOwner) {
+  var divchkOwner = document.getElementById("divchkOwner");
+  var footer = document.querySelector("footer");
+  divchkOwner.style.display = chkOwner.checked ? "block" : "none";
+  // footer.style.position = chkOwner.checked ? "relative" : "absolute";
```

Comment:
remove the commented line.
use this library atom-beautify to fix indentation 
@walaamedhat

Reply:
done